src/Menu.js

```diff
@@ -0,0 +1,63 @@
+export default class Menu {
+  constructor() {
+    this.menu = document.createElement('div');
+    this.menu.classList.add('menu');
+    this.isShow = false;
+    this.tintScreen = document.createElement('div');
+    this.tintScreen.classList.add('tint__screen');
+    this.loadCategories();
+    this.tintScreen.addEventListener('click', this.hide.bind(this));
+  }
+
+  createMenu(res) {
+    let txt = `<ul>
+                    <a href="#home"><li>
```

Comment:
Wrong layout
`ul` should include only `li` tags and `li` could include `a` tags
Please run your application with [validatior](https://validator.w3.org/nu/?useragent=Validator.nu%2FLV+http%3A%2F%2Fvalidator.w3.org%2Fservices&acceptlanguage=&file=index.html#file)